logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/TransactionLogger.java

```diff
@@ -185,6 +185,14 @@ public static String getFlowContextAsync(TransactionLogger logger) {
   }
 
   /**
+   * Get string value of flow context for this
+   * @return string value of flow context
+   */
+  public String getFlowContextAsync(){
```

Comment:
Good point.
Changed it so it checks for null and returns null in that case. Added to documentation.